src/components/movieGenreList/MovieGenreList.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGenreFilms, fetchGenres } from '../../services/fetchingData';
+import MovieList from '../movieList/MovieList';
+import { cleanFilmsAction } from '../../modules/reducer';
+
+const MovieGenreList = ({ genre }) => {
```

Comment:
In some componetns you are using the nice approach with creating container components. If you want you can apply this approach more widely.